java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -1586,7 +1586,7 @@ public boolean isMainlineD() {
      * @param p the point
      * @param useRectangles use hit rectangle (false: use hit circles)
      * @param requireUnconnected (only hit disconnected connections)
-     * @return 
+     * @return value representing the point connection type
      * @since 7.4.?
```

Comment:
Indeed we are, but as this tag already was here and I didn't have the opportunity to fill in the author's ``?``  from Github Blame, we'll have to get back to this later. Glad that Findbugs isn't too strict about this